tests/conftest.py

```diff
@@ -335,3 +336,8 @@ def _builder(id, name, questions=[], section_id=DEFAULT_SECTION["id"]):
         }
 
     return _builder
+
+
+@pytest.fixture
+def autogenerated_user_uuid():
```

Comment:
This isn't a great fixture name in this context (probably copy / paste from the other repo). Maybe just `autogenerated_uuid` is better?